testtools/testcase.py

```diff
@@ -536,6 +542,9 @@ def expectThat(self, matchee, matcher, message='', verbose=False):
                 )
             )
             self.force_failure = True
+            return False, str(mismatch_error)
```

Comment:
I don't think there is any need for a tuple here: an Option makes more sense, which in idiomatic Python is just None-or-value. So lets just return mismatch_error.